python/torch_mlir_e2e_test/test_suite/nll_loss.py

```diff
@@ -133,6 +133,52 @@ def forward(self, x, y):
 def NllLossModule_ignore_index_out_of_bounds_basic(module, tu: TestUtils):
   module.forward(tu.rand(2, 3), tu.randint(2, low=0, high=3))
 
+class NllLossModule_2D(torch.nn.Module):
```

Comment:
If you just need the ODS, there's no need to add tests. Tests should be added in the same PR as the op lowering so that it is easier for reviewers to see the different edge cases being tested.

Reply:
fixed